8.cytopick_analysis/notebooks/find_prediction_images.ipynb

```diff
@@ -1,1914 +1,1911 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    image_out_dir_path.mkdir(parents=True, exist_ok=True)
</pre><p>Consider adding this line where you are declaring the <code>image_out_dir-path</code> to improve readability.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>change made</p>